cartoframes/core/managers/context_manager.py

```diff
@@ -46,13 +46,14 @@ def copy_to(self, source, schema, limit=None, retry_times=DEFAULT_RETRY_TIMES, d
         copy_query = self._get_copy_query(query, columns, limit, drop_the_geom_webmercator)
         return self._copy_to(copy_query, columns, retry_times)
 
-    def copy_from(self, cdf, table_name, if_exists):
+    def copy_from(self, cdf, table_name, if_exists, _dry_run=False):
```

Comment:
Should we rename this to `verbose`, `log` or something like that? because it's not actually a dry_run like in the geocoding, for example. I would try to avoid underscored params too.

Reply:
it'd be better to use some kind of loggin util to do this, yes. Let me think about it.